browser/components/BrowserContentHandler.jsm

```diff
@@ -238,16 +243,12 @@ function openBrowserWindow(
       Ci.nsIToolkitProfileService
     );
     if (isStartup && pService.createdAlternateProfile) {
-      // Suppress showing about:newinstall for profiles for Replay which
-      // may be installed next to other versions of Firefox and _should_
-      // use a new profile
-
-      // let url = NEWINSTALL_PAGE;
-      // if (Array.isArray(urlOrUrlList)) {
-      //   urlOrUrlList.unshift(url);
-      // } else {
-      //   urlOrUrlList = [url, urlOrUrlList];
-      // }
+      let url = getNewInstallPage();
+      if (Array.isArray(urlOrUrlList)) {
+        urlOrUrlList.unshift(url);
+      } else {
+        urlOrUrlList = [url, urlOrUrlList];
+      }
```

Comment:
this looks pretty similar to what we had before. Why are we commenting it out?

Reply:
I'm effectively restoring previous logic but with the new welcome page rather than `about:newinstall`

Reply:
The result being if the browser created an alt profile (unlikely now but probably still possible), the user will get the welcome page rather than the warning "about page".